autosklearn/estimators.py

```diff
@@ -527,8 +528,12 @@ def predict_proba(self, X, batch_size=None, n_jobs=1):
             The predicted class probabilities.
 
         """
-        return super().predict_proba(
+        pred_proba = super().predict_proba(
             X, batch_size=batch_size, n_jobs=n_jobs)
+        assert np.allclose(np.sum(pred_proba, axis=1),
+                           np.ones_like(pred_proba[:, 0])),\
+        "prediction probability does not sum up to 1!"
```

Comment:
Could you please change the formatting to
```
assert (
    np.allclose(
        np.sum(pred_proba, axis=1),
        np.ones_like(pred_proba[:, 0])),
    ), "prediction probability does not sum up to 1!"

Reply:
Yep, I will make that modification. Currently some unittests fail due to this line. I'm working on the fix and I will remove [WIP] as soon as it is done.